docs/src/network_analysis/odes.md

```diff
@@ -0,0 +1,359 @@
+# [Decomposing the Reaction Network ODEs](@id network_analysis_odes)
+
+In this tutorial we will discuss the specific mathematical 
+structure of the [ODEs that arise from the mass-action dynamics](@ref math_models_in_catalyst_rre_odes)
+of chemical reaction networks, and decompose them as a product
+of matrices that describe the network.  A complete summary of 
+the exported functions is given in the API section [Network Analysis and Representations](@ref api_network_analysis). Please consult Feinberg's *Foundations of Chemical Reaction
+Network Theory*[^1] for more discussion about the concepts on this page.
+
+Note, currently API functions for network analysis and conservation law analysis
+do not work with constant species (currently only generated by SBMLToolkit).
+
+## [Network representation of the Repressilator `ReactionSystem`](@id network_analysis_repressilator_representation)
+We first load Catalyst and construct our model of the repressilator
+```@example s1
+using Catalyst
+repressilator = @reaction_network Repressilator begin
+       hillr(P₃,α,K,n), ∅ --> m₁
+       hillr(P₁,α,K,n), ∅ --> m₂
+       hillr(P₂,α,K,n), ∅ --> m₃
+       (δ,γ), m₁ <--> ∅
+       (δ,γ), m₂ <--> ∅
+       (δ,γ), m₃ <--> ∅
+       β, m₁ --> m₁ + P₁
+       β, m₂ --> m₂ + P₂
+       β, m₃ --> m₃ + P₃
+       μ, P₁ --> ∅
+       μ, P₂ --> ∅
+       μ, P₃ --> ∅
+end
+```
+In the [Introduction to Catalyst](@ref introduction_to_catalyst)
+tutorial we showed how the above network could be visualized as a
+species-reaction graph. There, species are represented by the nodes of the graph
+and edges show the reactions in which a given species is a substrate or product.
+```julia
+g = Graph(repressilator)
+```
+![Repressilator solution](../assets/repressilator.svg)
+
+We also showed in the [Introduction to Catalyst](@ref introduction_to_catalyst) tutorial that
+the reaction rate equation ODE model for the repressilator is
```

Comment:
first mention of the RRE, maybe change this to "reaction rate equation (RRE)" and use rre later on?